tests/unit/states/test_file.py

```diff
@@ -1174,6 +1183,7 @@ def test_prepend(self):
         '''
         Test to ensure that some text appears at the beginning of a file.
         '''
```

Comment:
Does this work on linux?

Reply:
good catch.

Reply:
The tests looked like they passed, so I merged it.

Reply:
Oh, right. They would pass because 'c:\\etc' wont ever exist on linux. It's still buggy though so I'll clean it up in another PR (after making sure test_prepend actually passes now).